src/Symfony/Component/Validator/Attribute/InitializesOptions.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class InitializesOptions
```

Comment:
why do we need this attribute ? We already have the `HasNamedArguments` attribute on the constructor which says that the constraint supports using it as named arguments, and we already deprecated passing options in the old way in most core constraints after migrating them to this attribute.

Reply:
My initial idea was to completely deprecate the `$options` argument of the base `Constraint` class. But it turns out that this breaks when named arguments are used. I have removed the attribute again and only deprecated passing anything else than `null` for the `$options` argument.